preshed/bloom.pyx

```diff
@@ -3,13 +3,15 @@
 #
 from murmurhash.mrmr cimport hash128_x86
 import math
-from array import array
+import struct
 
 try:
     import copy_reg
 except ImportError:
     import copyreg as copy_reg
 
+# TODO better way to declare constant?
+cdef key_t KEY_BITS = 8 * sizeof(key_t)
```

Comment:
Is there a better way to declare a constant? Should I use an anonymous enum [like the Cython docs say](https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#:~:text=but%20you%20can%20use%20an%20anonymous%20enum%20declaration%20for%20this%20purpose)?

Reply:
Yes, that is better than a mutable variable.